spec/dummy/app/views/components/modal.slim

```diff
@@ -0,0 +1,6 @@
+button#show Show
+
+= ui_component 'modal', \
```

Comment:
Should the modal really know about the trigger? Shouldn't a link/button trigger the modal instead?